waterlog/src/Globals.js

```diff
@@ -1,3 +1,3 @@
 export const Globals = {
-	API_URL: 'http://api.iot.retrotest.co.za'
+    API_URL: 'https://localhost:44382'
```

Comment:
I don't think this is what you want. I understand having to change it on the local machine for testing. Maybe consider using environment variables, that way the code is the same but can be changed depending on where it is built. 

https://medium.com/@trekinbami/using-environment-variables-in-react-6b0a99d83cf5

Reply:
I have tried environment variables, couldn't get it to work, will keep working on them.